Gemfile

```diff
@@ -9,3 +9,5 @@ gem 'workarea-testing', path: 'testing'
 gem 'listen'
 gem 'bundler-audit'
 gem 'rubocop'
+gem 'rails-decorators', git: 'https://github.com/workarea-commerce/rails-decorators.git'
+gem 'teaspoon', git: 'https://github.com/jtapia/teaspoon.git', branch: 'chore/update-rails-6'
```

Comment:
Ugh

Reply:
Hoping that gets merged in, but if not, we can just specify this in the base `Gemfile`. It doesn't look like any builds are writing teaspoon tests.